src/sentry/models/rule.py

```diff
@@ -24,6 +25,18 @@ class RuleStatus:
     DELETION_IN_PROGRESS = 3
 
 
+class RuleType(IntEnum):
+    ISSUE = 0
+    CRON_MONITOR_RULE = 1
+
+    @classmethod
+    def as_choices(cls) -> Sequence[Tuple[int, str]]:
```

Comment:
aren't metric alerts another rule type? or is that a whole separate thing

Reply:
lol told you this would be confusing @wedamija

Yeah they are a totally different thing

Reply:
add a comment? :D

Reply:
lol, we think we've come up with a clearer name